spring-cloud-open-service-broker-core/src/main/java/org/springframework/cloud/servicebroker/model/binding/CreateServiceInstanceAppBindingResponse.java

```diff
@@ -132,7 +141,7 @@ public final boolean canEqual(Object other) {
 
 	@Override
 	public final int hashCode() {
-		return Objects.hash(super.hashCode(), credentials, syslogDrainUrl, volumeMounts);
+		return Objects.hash(super.hashCode(), credentials, syslogDrainUrl, volumeMounts, endpoints);
 	}
 
 	@Override
```

Comment:
Shall we include the endpoints to the `toString` method too?

Reply:
yes. good catch!

Reply:
I've updated the `toString` method in the latest commit